components/brave_rewards/resources/tip/components/siteBanner.tsx

```diff
@@ -134,7 +134,7 @@ class Banner extends React.Component<Props, State> {
     } else if (mediaMetaData.mediaType === 'reddit') {
       return `u/${mediaMetaData.userName}`
     } else if (mediaMetaData.mediaType === 'github') {
-      return `@${mediaMetaData.userName}`
+      return `@${mediaMetaData.publisherScreenName}`
```

Comment:
I see that in `tipMediaUser.tsx` we use `'@' + mediaMetaData.publisherName`, where here we use `mediaMetaData.publisherScreenName`. I would suggest that we do the same in both places and this way you don't need to introduce new parameter

Reply:
I think that's just a bug in `tipMediaUser.tsx` (which is leading to issue #4 that you noticed below). The issue is that the Github tip dialog wants to show both the screen name (e.g. `emerick`) and the publisher name (e.g., `Emerick Rogul`). For some reason, the Twitter integration only ever showed one name so we were good.